examples/multilabel_classification/1-quickstart/plot_tutorial_multilabel_classification.py

```diff
@@ -102,16 +102,16 @@
 # Bernstein and Waudby-Smith–Ramdas).
 # The two methods give two different guarantees on the risk:
 #
-# * RCPS: :math:`P(R(\mathcal{T}_{\hat{\lambda}})\leq\alpha)\geq 1-\delta`
-#   where :math:`R(\mathcal{T}_{\hat{\lambda}})`
-#   is the risk we want to control and :math:`\alpha` is the desired risk
+# * RCPS: ``P(R(𝒯ₓ̂ₗ) ≤ α) ≥ 1 - δ``
+#   where ``R(𝒯ₓ̂ₗ)``
+#   is the risk we want to control and α is the desired risk
 #
-# * CRC: :math:`\mathbb{E}\left[L_{n+1}(\hat{\lambda})\right] \leq \alpha`
-#   where :math:`L_{n+1}(\hat{\lambda})` is the risk of a new observation and
```

Comment:
That one is broken. You can use this tool (an online reStructured editor) to translate : https://feat.dlup.link/rsted

Reply:
Done!